app/code/Magento/MediaContentApi/Model/Composite/GetAssetIdsByContentField.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContentApi\Model\Composite;
+
+use Magento\Framework\Exception\InvalidArgumentException;
+use Magento\MediaContentApi\Api\GetAssetIdsByContentFieldInterface as GetAssetIdsByContentFieldApiInterface;
+use Magento\MediaContentApi\Model\GetAssetIdsByContentFieldInterface;
+
+/**
+ * Class responsible to return Asset ids by content field
+ */
+class GetAssetIdsByContentField implements GetAssetIdsByContentFieldApiInterface
+{
+    /**
+     * @var GetAssetIdsByContentFieldInterface[]
+     */
+    private $fieldHandlers;
+
+    /**
+     * GetAssetIdsByContentField constructor.
+     *
+     * @param array $fieldHandlers
+     */
+    public function __construct($fieldHandlers = [])
+    {
+        $this->fieldHandlers = $fieldHandlers;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function execute(string $field, string $value): array
+    {
+        if (!array_key_exists($field, $this->fieldHandlers)) {
+            throw new InvalidArgumentException(__('The field argument is invalid.'));
+        }
+        $ids = [];
+        /** @var GetAssetIdsByContentFieldInterface $fieldHandlers */
```

Comment:
unnecessary annotation (the type is already declared on property)

Reply:
I will actually update the property, that is not correct, it is a multidimensional array, instead of an array of GetAssetIdsByContentFieldInterface. I've added the notation because PhpStorm doesn't recognize the object.